samples/simpleasset/mocks/statequeryiterator.go

```diff
@@ -0,0 +1,232 @@
+// Code generated by counterfeiter. DO NOT EDIT.
```

Comment:
Do you need to check these `mocks` files in? I thought you were writing a `Makefile` to import them on the fly from Fabric Interop CC folder?